src/EnergyPlus/DElightManagerF.cc

```diff
@@ -313,9 +313,9 @@ namespace DElightManagerF {
                     // Count the number of opaque surfaces bounding the current zone
                     iNumOpaqueSurfs = 0;
                     for (int spaceNum : zn.spaceIndexes) {
-                        auto &thisSpace = state.dataHeatBal->space(spaceNum);
+                        auto const &thisSpace = state.dataHeatBal->space(spaceNum);
                         for (int isurf = thisSpace.HTSurfaceFirst; isurf <= thisSpace.HTSurfaceLast; ++isurf) {
-                            auto &surf(state.dataSurface->Surface(isurf));
+                            auto const &surf(state.dataSurface->Surface(isurf));
```

Comment:
[src/EnergyPlus/DElightManagerF.cc:316]:(style),[constVariable],Variable 'thisSpace' can be declared as reference to const
[src/EnergyPlus/DElightManagerF.cc:318]:(style),[constVariable],Variable 'surf' can be declared as reference to const